src/open_inwoner/openzaak/constants.py

```diff
@@ -7,3 +7,13 @@ class StatusIndicators(models.TextChoices):
     warning = "warning", _("Warning")
     failure = "failure", _("Failure")
     success = "success", _("Success")
+
+
+class ZaakTitleDisplayChoices(models.TextChoices):
+    zaak_titel = "zaaktype_title", _("The title of the case (`zaak.omschrijving`)")
```

Comment:
```suggestion
    zaak_omschrijving = "zaak_omschrijving", _("The description of the case (`zaak.omschrijving`)")
```

or even:
```
    zaak_omschrijving = "zaak_omschrijving", _("The description of the particular case (`zaak.omschrijving`)")
``` 
to bring out the contrast with the zaak type.

The option is to use the `omschrijving` of the particular `zaak` as `title` for the `zaak`, right?

Reply:
:facepalm: yup.